net-firewall/nftables/nftables-9999.ebuild

```diff
@@ -206,12 +207,22 @@ pkg_postinst() {
 	fi
 
 	if has_version 'sys-apps/systemd'; then
+		local _replacing_version=
+		for _replacing_version in ${REPLACING_VERSIONS}; do
+			if ver_test "${_replacing_version}" -lt "1.1.1-r1"; then
```

Comment:
This looks like a good fit for the `ver_replacing` function from eapi9-ver.eclass.